### DISCUSSION TITLE
Complete data loss and restoration on loop.

### DISCUSSION THREAD

**Discussion by aaronjconway** [supabase/supabase#27578] [Questions]
I'm lucky this was just a side project - I let it pause and came back to no database and the project on infinite restoration - it has been this way for several days. I think if that's a possibility, for better expectations just don't have a free tier or make the consequences even worse so people don't let it pause. I understand it's completely my fault that I had no backups. Just brutal.

**Comment by GaryAustin1** [comment]
Contact support.  They will bring it back up

**Reply by aaronjconway** [reply]
I received an email saying there is no guaranteed response yesterday  - it's the free tier - I completely understand. It's a great product,, lesson learned.
